### DISCUSSION TITLE
Quake-style hotkey window with multiple monitors

### DISCUSSION THREAD

**Discussion by svallory** [wezterm/wezterm#4302] [Q&A]
Hi guys!

Is  it possible to make the wezterm window appear on the screen where the mouse currently is?

I configured a quake-style hotkey wezterm using Kerabiner ([here's the config if anyone wants it](https://gist.github.com/svallory/0cc08750e5ae837adad3ee3dde3599c9)), but I have a 3-monitor setup and I would like to make the window appear on the screen I'm currently working on (usually, there's where the mouse is or, for sure, where the frontmost application is).

I tried a lot of things and could get the window to show up (if it's not maximized or in fullscreen) on the correct window. But I can't get it to appear on the correct position.

**Comment by Hugo-Persson** [comment]
Was any progress made on this?
